### PR TITLE
Add launcher command section

### PR DESCRIPTION
**Pull request by Jessie212**
## Description

> Is this change a fix, improvement, new feature, refactoring, or other?

Improvement.

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)

Changes were made to documentation only.

> How would you describe this change to a non-technical end user or system administrator?

Add launch command section to Trino deployment page.

## Documentation

( ) No documentation is needed.
(x) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(x) No release notes entries required.
( ) Release notes entries required with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: docs/src/main/sphinx/installation/deployment.rst [hunk 1]
```diff
@@ -281,12 +281,94 @@ contents to mount the ``jmx`` connector as the ``jmx`` catalog:
...
+   jvm_config      = /etc/sep/jvm.config
+   jvm_options     = []
+   launcher_config = /opt/starburst/bin/launcher.properties
+   launcher_log    = /opt/starburst/var/log/launcher.log
+   log_levels      = /etc/sep/log.properties
+   log_levels_set  = False
+   node_config     = /etc/sep/node.properties
+   pid_file        = /opt/starburst/var/run/launcher.pid
+   properties      = {'node.environment': 'demo', 'node.id': 'ffffffff-ffff-ffff-ffff-ffffffffffff'}
+   server_log      = /opt/starburst/var/log/server.log
```
Comment (martint) [inline docs/src/main/sphinx/installation/deployment.rst:339]: Use /etc/trino, /opt/trino, etc.

File: docs/src/main/sphinx/installation/deployment.rst [hunk 2]
```diff
@@ -281,12 +281,94 @@ contents to mount the ``jmx`` connector as the ``jmx`` catalog:
...
+accepts six commands:
+
+.. _using-launcher:
+
+Using the ``launcher`` command
+------------------------------
+
+The installation provides a ``bin/launcher`` utility, which requires Python in
+the ``PATH``. The utility can be used manually or as a daemon startup script. It
+accepts six commands:
```
Comment (martint) [inline docs/src/main/sphinx/installation/deployment.rst:295]: > *these* commands

So that we don't have to keep it in sync with the list below.

**Review by mosabua — CHANGES_REQUESTED** [review]
Needs reorder and fix ups

File: docs/src/main/sphinx/installation/deployment.rst [hunk 3]
```diff
@@ -286,7 +286,75 @@ See :doc:`/connector` for more information about configuring connectors.
 Running Trino
 --------------
 
-The installation directory contains the launcher script in ``bin/launcher``.
+The installation provides a ``bin/launcher`` utility, which requires Python in
```
Comment (mosabua) [inline docs/src/main/sphinx/installation/deployment.rst:289]: lets call it "script" .. not "utility"

File: docs/src/main/sphinx/installation/deployment.rst [hunk 4]
```diff
@@ -286,7 +286,75 @@ See :doc:`/connector` for more information about configuring connectors.
...
+   launcher_log    = /opt/trino/var/log/launcher.log
+   log_levels      = /etc/trino/log.properties
+   log_levels_set  = False
+   node_config     = /etc/trino/node.properties
+   pid_file        = /opt/trino/var/run/launcher.pid
+   properties      = {'node.environment': 'demo', 'node.id': 'ffffffff-ffff-ffff-ffff-ffffffffffff'}
+   server_log      = /opt/trino/var/log/server.log
+   verbose         = True
+
+A number of additional options allow you to specify non-default configuration
```
Comment (mosabua) [inline docs/src/main/sphinx/installation/deployment.rst:338]: Lets move this up and have the -v stuff afterwards

File: docs/src/main/sphinx/installation/deployment.rst [hunk 5]
```diff
@@ -286,7 +286,75 @@ See :doc:`/connector` for more information about configuring connectors.
...
+   --pid-file=FILE           Defaults to DATA_DIR/var/run/launcher.pid
+   --launcher-log-file=FILE  Defaults to DATA_DIR/var/log/launcher.log (only in
+                             daemon mode)
+   --server-log-file=FILE    Defaults to DATA_DIR/var/log/server.log (only in
+                             daemon mode)
+   -J OPT                    Set a JVM option
+   -D NAME=VALUE             Set a Java system property
+
+The ``-v`` or ``--verbose`` option for each command prepends the server's
+current settings before the command's usual output, much like the following
```
Comment (mosabua) [inline docs/src/main/sphinx/installation/deployment.rst:337]:
```suggestion
current settings before the command's usual output. For example, you can start a Trino installation in ``/opt/trino`` and get out similar to the following:
```

File: docs/src/main/sphinx/installation/deployment.rst [hunk 6]
```diff
@@ -286,7 +286,75 @@ See :doc:`/connector` for more information about configuring connectors.
...
+examples:
+
+.. code-block:: text
+
+   config_path     = /etc/trino/config.properties
```
Comment (mosabua) [inline docs/src/main/sphinx/installation/deployment.rst:342]: Replace this snippet with

```
$ cd /opt/trino
$ bin/launcher -v status
config_path     = /opt/trino/etc/config.properties
data_dir        = /opt/trino
etc_dir         = /opt/trino/etc
install_path    = /opt/trino
jvm_config      = /opt/trino/etc/jvm.config
jvm_options     = []
launcher_config = /opt/trino/bin/launcher.properties
launcher_log    = /opt/trino/var/log/launcher.log
log_levels      = /opt/trino/etc/log.properties
log_levels_set  = False
node_config     = /opt/trino/etc/node.properties
pid_file        = /opt/trino/var/run/launcher.pid
properties      = {'node.environment': 'mm_demo'}
server_log      = /opt/trino/var/log/server.log
verbose         = True

Not running
```

**Review by mosabua — COMMENTED** [review]
Clarified some more stuff in the comments.

**Review by mosabua — COMMENTED** [review]
Some of the stuff later down should be cleaned up as well .. ping me if you want to fine tune together @Jessie212

**Review by Ordinant — APPROVED** [review]
Looks fine to me, but I didn't check line wrapping issues.

**Review by mosabua — APPROVED** [review]
Nice addition.

**Comment by mosabua** [comment]
This is ready now @martint  .. for merge right after the release is done ;-)

**Comment by mosabua** [comment]
Build failure is wrong. I just built locally.

**Pull request merged**